x-pack/legacy/plugins/apm/public/index.scss

```diff
@@ -12,4 +12,5 @@
 
 .apmReactRoot {
   overflow-x: auto;
+  height: 100%;
```

Comment:
This was needed to vertically centralize the message `Fetching trace...` in the transition page.